rtengine/ipwavelet.cc

```diff
@@ -629,13 +596,13 @@ void ImProcFunctions::ip_wavelet(LabImage * lab, LabImage * dst, int kall, const
 
     // Calculate number of tiles. If less than omp_get_max_threads(), then limit num_threads to number of tiles
     if (options.rgbDenoiseThreadLimit > 0) {
-        maxnumberofthreadsforwavelet = min(max(options.rgbDenoiseThreadLimit / 2, 1), maxnumberofthreadsforwavelet);
+        maxnumberofthreadsforwavelet = rtengine::min(rtengine::max(options.rgbDenoiseThreadLimit / 2, 1), maxnumberofthreadsforwavelet);
```

Comment:
That rings a bell: don't we have something like `LIM<>()` for this pattern?

Reply:
@Floessie I missed your comments. Wouldn't have merged then. Anyway. I will fix this stuff

Reply:
Done